jupyter/minimal/ubi8-python-3.8/Pipfile

```diff
@@ -0,0 +1,26 @@
+
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+# Base
+jupyterlab = "~=3.5.3"
+
+# Useful extensions
+jupyter-server = "~=2.1.0"
+jupyter-server-proxy = "~=3.2.2"
+jupyter-server-terminals = "~=0.4.4"
+jupyterlab-git = "~=0.41.0"
+nbdime = "~=3.1.1"
+nbgitpuller = "~=1.1.1"
+
+# Parent image requirements to maintain cohesion
+thamos = "~=1.29.1"
+wheel = "~=0.38.4"
```

Comment:
in hindsight, 
if we are installing these bits here, we can remove the [base](./base) layer completely. 
Right? it would make one  less image.

Reply:
Base may be required for the runtime images, to use in the pipelines. Those images will be identical to the ootb notebooks, but without all the Jupyter stuff. We cannot use directly the Jupyter images as runtimes because of some incompatibilities with the Elyra process, when its bootstrap parses the notebooks.

Reply:
Right, moreover we use the base to build the CUDA layer.